test/packages/custom_ilm_policy/changelog.yml

```
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: "0.0.1"
+  changes:
+    - description: Initial draft of the package
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/1 # FIXME Replace with the real PR link
```

In the changelog of these test packages, should we update the links to be the PRs where they have been added ?

```suggestion
      link: https://github.com/elastic/integrations/pull/413
```